notebooks/official/training/multi_node_ddp_gloo_vertex_training_with_custom_container.ipynb

```diff
@@ -1301,16 +1233,22 @@
       },
       "outputs": [],
       "source": [
-        "# Set this to true only if you'd like to delete your bucket\n",
+        "# Set this to true only if you'd like to delete\n",
```

Comment:
why remove "your bucket"?  "Delete" is a transitive verb and requires a direct object. In this case, it would be "your bucket."